manager/manager.go

```diff
@@ -69,73 +72,95 @@ func NewTransport(dc dispatch.DispatcherConfig) http.RoundTripper {
 // chrysom client listener
 func (m *Manager) Update(items []argus.Item) {
 	recentMap := make(map[string]model.Norn)
-	newNorns := []nornDispatcher{}
-	oldNorns := []dispatch.D{}
+	newNorns := []filterDispatcher{}
+	oldNorns := []dispatch.F{}
 
 	transport := NewTransport(m.dispatcherConfig)
```

Comment:
We shouldn't make a new transport with every update - the configuration is always the same.  The sender should be created with the manager, and then we never have to create it again.